tests/v3/covid-19/api_gov.spec.js

```diff
@@ -6,7 +6,7 @@ const { should } = require('chai');
 
 chai.use(chaiHttp);
 
-const countries = ['Austria', 'Canada', 'Italy', 'Germany', 'Switzerland', 'Nigeria', 'India', 'Vietnam', 'New Zealand', 'Colombia', 'South Africa'].sort();
+const countries = ['Austria', 'Canada', 'Italy', 'Germany', 'Switzerland', 'Nigeria', 'India', 'Vietnam', 'New Zealand', 'Colombia', 'South Africa', 'UK'].sort();
```

Comment:
Are you going to add any UK specific tests?

Reply:
> Are you going to add any UK specific tests? 

no, should I?

Reply:
Yeah, there are tests for every other one

Reply:
good point